src/ObjectGraph.es6.js

```diff
@@ -182,21 +184,25 @@ ObjectGraph.prototype.rewriteName = function(name) {
 
 // Visit the prototype of o, given its dataMap.
 ObjectGraph.prototype.visitPrototype = function(o, dataMap) {
+  if ( this.protos[uid.getId(o)] !== undefined ) {
+    console.log(uid.getId(o));
```

Comment:
The logging is for debug. I will remove this line.

maybeSkip only checks if the id exists in datamap. VisitPrototype is enqueued after the properties of id are stored in datamap, so maybeSkip does not work here.

After add the regression check in id-js. The 'repeated store proto' bug is solved. So I believe the bug is resulted from Storage items and their proto returning same +UID. I think it is okay to remove these lines now.